### PR TITLE
Fixes borers in mobs getting blasted by weather

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
While within a mob, you don't get affected by weather as a borer.

## Why It's Good For The Game
Being killed while within a host by ash storms not good

## Proof Of Testing
Tested borers dying outside of bodies, and not dying inside, works

## Changelog

:cl:
fix: Borers no longer get wasted by ash storms while inside a host
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
Literal IDED PR

**Comment by destrucktoid** [comment]
I've had this problem before as borer as well, a good change.

File: modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm [hunk 1]
```diff
@@ -572,6 +573,7 @@
 	var/logging_text = "[key_name(cortical_owner)] went into [key_name(cortical_owner.human_host)] at [loc_name(human_turftwo)]"
 	cortical_owner.log_message(logging_text, LOG_GAME)
 	cortical_owner.human_host.log_message(logging_text, LOG_GAME)
+	ADD_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host")
```
Comment (nevimer) [inline modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm:576]:
```suggestion
	ADD_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host") // BUBBER EDIT - WEATHER IMMUNE
```

File: modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm [hunk 2]
```diff
@@ -509,6 +509,7 @@
 			borer_organ.Remove(cortical_owner.human_host)
 		cortical_owner.forceMove(human_turfone)
 		cortical_owner.human_host = null
+		REMOVE_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host")
```
Comment (nevimer) [inline modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm:512]:
```suggestion
		REMOVE_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host") // BUBBER EDIT - WEATHER IMMUNE
```

File: modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm (hunk 1 above)
Comment (Arturlang) [reply modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm]: Coulda pinged me on dsicord or given me more than 3 days of time to do this but thanks

**Review by Majkl-J — APPROVED** [review]
This could probably be taken upstream, but looks more than ready to merge anyhow. Feel free to mirror this to skyrat whenever you feel like it would be convenient

**Pull request merged**